src/include/ipc/header.h

```diff
@@ -151,6 +152,21 @@
 
 /** @} */
 
+/** \name DSP Command: Probes
+ *  @{
+ */
+
+#define SOF_IPC_PROBE_INIT			SOF_CMD_TYPE(0x001)
+#define SOF_IPC_PROBE_DEINIT			SOF_CMD_TYPE(0x002)
+#define SOF_IPC_PROBE_DMA_SET			SOF_CMD_TYPE(0x003)
+#define SOF_IPC_PROBE_DMA_GET			SOF_CMD_TYPE(0x004)
+#define SOF_IPC_PROBE_DMA_DETACH		SOF_CMD_TYPE(0x005)
+#define SOF_IPC_PROBE_POINT_SET			SOF_CMD_TYPE(0x006)
+#define SOF_IPC_PROBE_POINT_GET			SOF_CMD_TYPE(0x007)
+#define SOF_IPC_PROBE_POINT_REMOVE		SOF_CMD_TYPE(0x008)
```

Comment:
same comments as for the kernel side, it'd help to have comments to figure out why things are not symmetrical.
You have init-deinit, DMA-set/get, point-set/get, so what are 'detach' and 'remove' doing?

Reply:
Can you pretty please *EXPLAIN* the definition of these IPCs and the expected flow?
I made the same remark last week and there's just no update.

Reply:
@plbossart Flow is described in first commit message and PR description, for review and comments, but I agree that it should also be documented within the code, for future reference. We'll update these once we get everything up and running.

Reply:
ok, thanks